### PR TITLE
Upgrade dependencies, fix breaking change in rollup

### PR DESCRIPTION
**Pull request by alexdevero**
Updates all project dependencies to their latest versions (16th July 2017). Then, adds fix for breaking change in rollup dependency suggested by @black-snow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,22 +33,22 @@
     "fs-jetpack": "^1.0.0"
   },
   "devDependencies": {
-    "chai": "^4.0.1",
-    "electron": "^1.6.11",
-    "electron-builder": "^19.5.1",
+    "chai": "^4.1.0",
+    "electron": "^1.7.4",
```
Comment (black-snow) [inline package.json]: 1.7.4 is still in beta

**Comment by black-snow** [comment]
Looks okay 'cept for the electron beta version. npm shouldn't update to a beta version but it recently did for me ... dunno if that's intended. If that's the case the caret (^) would update to 1.7.4 anyway.
There were quite some changes in electron-builder but no breaking ones so that should be okay.

**Comment by alexdevero** [comment]
@black-snow I used yarn to install and upgrade the dependencies. Maybe, this is the cause. Do you want to lower the version to the latest stable, v1.6.11?

**Comment by black-snow** [comment]
If I remember it correctly npm and yarn both installed 1.7.4 beta when it's 1.6.x with caret. Dunno.

Anyway, yes, I'd recommend setting it to latest stable. Also there's that one spot in the README where we [mention the electron version](https://github.com/szwacz/electron-boilerplate/blob/master/README.md#upgrading-electron-version) (currently 1.6.6) - please update that, too ;)

Thanks a lot for your PR.

**Comment by alexdevero** [comment]
Issues are fixed. And, you are welcome. I love to help and contribute, no matter how big or small the help or contribution is.

**Pull request merged**